daemon/mgr/container.go

```diff
@@ -176,6 +176,10 @@ func (mgr *ContainerManager) CreateExec(ctx context.Context, name string, config
 		return "", err
 	}
 
+	if !c.IsRunning() {
+		return "", fmt.Errorf("container: %s is not running", c.ID())
```

Comment:
Good catch.
To be honest, I am wondering if we should add `:` to be `container: asdfghjk is not running`.
Actually I prefer `container asdfghj is not running` without `:`.

In addition, could you move `execid := randomid.Generate()` right before `execid := randomid.Generate()`. It seems to be a little bit more clear.

Reply:
Yes. It makes senses. @allencloud . Change is ready. PTAL. Thanks